02-Calculator/Calculator.js

```diff
@@ -3,14 +3,22 @@ import {MyMath} from "../01-MyMath/MyMath.js";
 export class Calculator {
 
     constructor(numpad, outputCalculation, outputSolution) {
+        this.numpad = numpad;
+        this.outputCalculation = outputCalculation;
+        this.outputSolution = outputSolution;
+        this.setupNumPad();        
     }
 
     setupNumPad() { 
+        var listofallnumbers = this.numpad.children;
+        for (let element of listofallnumbers) {
+            element.addEventListener('click', this.onButtonClick.bind(this, listofallnumbers))
```

Comment:
`this.onButtonClick.bind(this, listofallnumbers)` führt dazu, dass `symbol` in `onButtonClick` den wert von `listofallnumbers` hat. Ich denke nicht, dass das gewollt ist. Du willst ja den Wert der auf der Taste steht erhalten.
```suggestion
            element.addEventListener('click', this.onButtonClick.bind(this, element.innerHTML))
```

Reply:
Super, danke dir!